src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1579,12 +1596,18 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
 
         private async Task InvokeValidationEndpointAsync() {
             OpenIdConnectMessage request;
-
+            
+            // Note: The updated RFC standard says nothing about allowing GET requests beyond
```

Comment:
Hm. This may be a more in-depth discussion to have (or a comment submission on the spec itself), because the RFC briefly mentions it, but that might be left over from previous revisions, and the previous revisions said it was up to the implementation if GET was supported due to server logs and such. I'd be okay with removing GET support on this endpoint, but wanted to leave it in until reviewed.